### PR TITLE
[FLINK-6256] Fix outputTag variable name in Side Output docs

### PR DESCRIPTION
**Pull request by zentol**
@kl0u Is this the issue of the JIRA or was there something else?

### CODE REVIEW

**Comment by kl0u** [comment]
@zentol Yes, this is the main issue, but you could also use the same name for the output tags between Java and Scala. Java uses "side-output" while Scala uses "string-side-output".

**Comment by zentol** [comment]
@kl0u done; i went with "side-output".

File: docs/dev/stream/side_output.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ side output stream:
 
 {% highlight java %}
 // this needs to be an anonymous inner class, so that we can analyze the type
-OutputTag<String> outputTag = new OutputTag<String>("string-side-output") {};
+OutputTag<String> outputTag = new OutputTag<String>("side-output") {};
```
Comment (kl0u) [inline docs/dev/stream/side_output.md]: You should update the Scala.

Comment (zentol) [inline docs/dev/stream/side_output.md]: done.

**Pull request closed without merging**